src/pathways/pathway/dto/list-pathway-users.dto.ts

```
@@ -0,0 +1,86 @@
+    description: 'Filter by active status in pathway',
+    example: true,
+  })
+  @Expose()
+  @IsOptional()
+  @IsBoolean()
+  status?: boolean;
+}
+
+class ListPathwayUsersSortDto {
+  @ApiPropertyOptional({
+    description: 'Column to sort by',
```

ESLint

[error] Replace `'Column·to·sort·by'` with `"Column·to·sort·by"`

(prettier/prettier)